streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1342,7 +1345,7 @@ private void addStandbyTasks(final long start) {
 
         final Map<TaskId, Set<TopicPartition>> newStandbyTasks = new HashMap<>();
 
-        log.info("{} Adding assigned standby tasks {}", logPrefix, partitionAssignor.activeTasks());
+        log.debug("{} Adding assigned standby tasks {}", logPrefix, partitionAssignor.activeTasks());
```

Comment:
Was reported at the mailing list. :) There is actually a PR for it in the mean time (was not there when I put the comment): #3362